amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2314,6 +2314,112 @@ paths:
+  '/holds/{holdId}/bulk-statuses':
+    get:
+      tags:
+        - holds
+      operationId: bulkStatuses
+      summary: Get bulk statuses
+      description: |
+        Gets bulk statuses for hold with id **holdId**.
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldBulkStatusPaging'
+        '400':
+          description: |
+            Invalid parameter: **holdId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to read **holdId**
+        '404':
+          description: "**holdId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/holds/{holdId}/bulk-statuses/{processId}':
+    get:
+      tags:
+        - holds
+      operationId: bulkStatus
+      summary: Get a bulk status
+      description: |
+        Gets a bulk status specified by **processId** for **holdId**.
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - $ref: '#/parameters/processId'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldBulkStatus'
+        '400':
+          description: |
+            Invalid parameter: **holdId** or **processId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to read **holdId**
+        '404':
+          description: "**holdId** or **processId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/holds/{holdId}/bulk':
+    post:
+      tags:
+        - holds
+      operationId: holdBulk
+      summary: Start the hold bulk process
+      description: |
+        Start the asynchronous bulk process of a hold with id **holdId** based on search query results.
+        
+        ```JSON
+        For example, the following JSON body start the bulk process to add search query results
+        as children of a hold.
+        
+        {
+          "query": {
+            "query": "SITE:swsdp and TYPE:content",
+            "language": "afts"
+          },
+          "op": "ADD"
+        }
+        ```
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - in: body
+          name: holdBulkOperation
+          description: Bulk operation.
+          required: true
+          schema:
+            $ref: '#/definitions/HoldBulkOperation'
+      responses:
+        '202':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldBulkStatus'
```

Comment:
IMO it doesn't make sense to reply with the full `HoldBulkStatus` here because we haven't started the bulk operation yet so most of the fields will have to be blank. In case of success we can pretty much only provide the `processId` and maybe the `totalItems` if that matters since that happens at validation time, before the bulk operation is started.

If the response is being cluttered by null fields such as `startTime`, `endTime`, `processedItems` etc. then I'd recommend having a separate model for this response.